src/goto-checker/bmc_util.h

```diff
@@ -251,7 +252,8 @@ void run_property_decider(
   "                              iteration are allowed to fail due to\n" \
   "                              complexity violations before the loop\n" \
   "                              gets blacklisted\n" \
-  " --graphml-witness filename   write the witness in GraphML format to filename\n" // NOLINT(*)
+  " --graphml-witness filename   write the witness in GraphML format to filename\n" /* NOLINT(*) */ \
+  " --symex-cache-dereferences    enable caching of repeated dereferences"
```

Comment:
As above: also add a `\` to the last line.

Reply:
Fixed, also extra space.

Reply:
Fixed.